% Template for PLoS
% Version 1.0 January 2009
%
% To compile to pdf, run:
% latex plos.template
% bibtex plos.template
% latex plos.template
% latex plos.template
% dvipdf plos.template

\documentclass[10pt]{article}

% amsmath package, useful for mathematical formulas
\usepackage{amsmath}
% amssymb package, useful for mathematical symbols
\usepackage{amssymb}

% graphicx package, useful for including eps and pdf graphics
% include graphics with the command \includegraphics
\usepackage{graphicx}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

\usepackage{color} 

% Use doublespacing - comment out for single spacing
%\usepackage{setspace} 
%\doublespacing


% Text layout
\topmargin 0.0cm
\oddsidemargin 0.5cm
\evensidemargin 0.5cm
\textwidth 16cm 
\textheight 21cm

% Bold the 'Figure #' in the caption and separate it with a period
% Captions will be left justified
\usepackage[labelfont=bf,labelsep=period,justification=raggedright]{caption}

% Use the PLoS provided bibtex style
\bibliographystyle{plos2009}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother


% Leave date blank
\date{}

\pagestyle{myheadings}
%% ** EDIT HERE **


%% ** EDIT HERE **
%% PLEASE INCLUDE ALL MACROS BELOW

%% END MACROS SECTION

\begin{document}

% Title must be 150 characters or less
\begin{flushleft}
{\Large
\textbf{openSNP - Crowdsourcing Genome Wide Association Studies}
}
% Insert Author names, affiliations and corresponding author email.
\\
Bastian Greshake$^{1,\ast}$, 
Philipp Bayer$^{2}$, 
Fabian Zimmer$^{3,}$
\\
\bf{1} Bastian Greshake Frankfurt am Main, Germany
\\
\bf{2} Philipp Bayer Gold Coast, Australia
\\
\bf{3} Fabian Zimmer M\"unster, Germany
\\
$\ast$ E-mail: info@opensnp.org
\end{flushleft}

% Please keep the abstract between 250 and 300 words
\section*{Abstract}
Genome wide association studies (GWAS) are a cheap and quick way to assess health risks by comparing Single Nucleotide Polymorphisms (SNPs) between groups of participants. Direct-To-Consumer Companies like 23andme offer their customers to sequence their SNPs alongside with an evaluation of the customer's genetic risks. However, the data 23andme and other companies generate is not accessible for other scientists, and withholds some information from their customers for various reasons. In this paper, we present an open approach to GWAS by introducing openSNP, a webpage which allows GWAS-customers to openly share their SNPs with scientists for free.  % misses survey


% Please keep the Author Summary between 150 and 200 words
% Use first person. PLoS ONE authors please skip this step. 
% Author Summary not valid for PLoS ONE submissions.   
\section*{Author Summary}

\section*{Introduction}



Paragraph1:


Investigate the link between genetic variation and phenotypic change
Important because of desease stuff.

Tool used: Genome Wide Association Studies.
--> Describe in one sentence.
--> 2005 first study
--> Today 1200 GWAS performed

genetic markers usually SNPs.
SNPs definition. Allele definition.
SNP can be linked to a phenotypic trait (around 5000 linked to desease)
 
Paragraph2:

How to obtain data for GWAS?
--> SNPChip, used in many different species
--> Now different companies for DTC genetic testing.
--> Test results provided to customer and raw data
--> Raw data can be used for GWAS, but closed



Paragraph3:

RAW data from 1000 of individuals could be used for multiple scientific
purposes
---> Large potential dataset Are the numbers available for dataset size used
in scientic studies? 2005 93, 2008 500, 2011 23andme 100000? Plot?

Resolution of the problem: Open Science
Create repository for sharing this information. openSNP


Paragraph4:

Describe openSNP:
Open Source, non-commercial
Open Data
Community driven --> Crowd sourcing


Parses data from 23andMe deCODEme....
--> Publications crawler (Maybe mention mendeley price?)
        --PLOS and Mendeley

Advantages for scientists: See paragraph 3

Advantages for customers: 
-->Make sense of the data
-->Contribute to the public domain



Genome Wide Association Studies (GWAS) are an easy and cheap way to find Single Nucleotide Polymorphisms (SNPs) which can be interesting because of their medical relevance. SNPs found through GWAS can be used to find candidate genes for a closer inspection or to predict disease risks. Genome Wide Association Studies make use of statistics to compare the alleles of patients to the alleles of healthy controls. By this the method does not allow to find causal differences but mere correlations. The first GWAS was published in 2005 and compared age-related macular degeneration in contrast to a healthy control group (doi:10.1126/science.1109557). Since the beginning the number of participants in those studies is rising and over 1200 GWAS have been performed (doi:10.1186/1471-2350-10-6.) and over 5000 SNPs have been linked to different diseases and traits in those studies %(http://www.genome.gov/page.cfm?pageid=26525384&clearquery=1#result_table).

Since 2006 companies like 23andMe, deCODEme or FamilyTreeDNA offer Direct-To-Consumer (DTC) genetic testing. Those companies use DNA micro arrays to screen for around 1 million SNPs spread over the human genome. In return customers get an analysis of the results, as well as a raw file that includes the SNP-IDs and their respective allele for the customer. In 2011 23andMe alone had over 100.000 customers\footnote{http://spittoon.23andme.com/2011/06/15/23andme-2011-state-of-the-database-address/} - the company recognizes the potential to perform GWAS with that amount of data by using surveys to ask their customers about traits and diseases. With the consent of the customer those data is used for association studies. 23andMe published several articles in which they replicate known findings but also find new associations for Parkinson's Disease \cite{Eriksson2010, Do2011}. Over 30,000 23andme-customers participated in those association studies.  

Although companies like 23andMe are willing to contribute to science it is not easy for individual scientists to access the data. This is mainly due to privacy concerns of the customers. Nevertheless there are individual customers who are willingly sharing their data. Most do so by uploading their data to their personal website or to open software repositories like GitHub. While this is makes it possible for scientists to access the data, it requires a lot of work to keep track of all new genotyping data that is available to the public. While projects like the SNPedia try to keep track of all the files, this still does not allow to perform GWAS, as the phenotypic information is not attached to the genetic information. Projects that attach the phenotype to the genetic information, like the Personal Genome Project, still don't allow for an easy re-use of the data. %WHY?  

A possible solution to this can be a community-driven platform that aggregates genetical and phenotypical information of people who are willing to share their data with the general public and have given their informed consent. We designed a survey to assess interest in such a crowd sourcing platform, in which we asked how many people would be willing to share their genetic and phenotypic information with the public. 

% Results and Discussion can be combined.
\section*{Results}

\subsection*{Survey on Sharing Genetic Information}
229 participants filled out the survey. Of these 229, 180 had a self-reported chromosomal sex of XY and 56 carried a self-reported chromosomal sex of XX. The mean age of the participants is 33 (SD = 11,29) and over 81.7 \% reported their ethnicity as caucasian. 39.7 \% of the participants are already customer of at least one DTC genetic testing company and further 30.1 \% of them plan to become one in the future. 29.7 \% don't plan to become a DTC customer. There is no significant difference in the usage of DTC companies between chromosomal sexes (Somers-d). 

67.7 \% of all participants would share their data with their DTC-company without any constraints, 25.8 \% would share if the company does not share the data with third parties. 6.6 \% of the participants would not share their data. There is no significant difference between sharing-habits between both chromosomal sexes (Somers-d). Those who are a customer of a DTC company or are planning to become one in the future are more likely to share their results, as compared to those who don't plan to get themselves genotyped (Somers-d). 

\subsection*{Introducing openSNP}
We've created openSNP, a website which allows users to upload their genotypings from the companies 23andme, deCODEme and Family Tree under the Creative Commons Zero-license. Users are encouraged to list as many phenotypes as possible due to a simple achievement-system similar to systems found in computer games. 

\section*{Discussion}
to-do list together with outlook of personalised medicine

% You may title this section "Methods" or "Models". 
% "Models" is not a valid title for PLoS ONE authors. However, PLoS ONE
% authors may use "Analysis" 
\section*{Materials and Methods}

% Do NOT remove this, even if you are not including acknowledgments
\section*{Acknowledgments}


%\section*{References}
% The bibtex filename
\bibliography{papers}

\section*{Figure Legends}
%\begin{figure}[!ht]
%\begin{center}
%%\includegraphics[width=4in]{figure_name.2.eps}
%\end{center}
%\caption{
%{\bf Bold the first sentence.}  Rest of figure 2  caption.  Caption 
%should be left justified, as specified by the options to the caption 
%package.
%}
%\label{Figure_label}
%\end{figure}


\section*{Tables}
%\begin{table}[!ht]
%\caption{
%\bf{Table title}}
%\begin{tabular}{|c|c|c|}
%table information
%\end{tabular}
%\begin{flushleft}Table caption
%\end{flushleft}
%\label{tab:label}
% \end{table}

\end{document}

