% Template for PLoS
% Version 1.0 January 2009
%
% To compile to pdf, run:
% latex plos.template
% bibtex plos.template
% latex plos.template
% latex plos.template
% dvipdf plos.template

\documentclass[10pt]{article}

% amsmath package, useful for mathematical formulas
\usepackage{amsmath}
% amssymb package, useful for mathematical symbols
\usepackage{amssymb}

% graphicx package, useful for including eps and pdf graphics
% include graphics with the command \includegraphics
\usepackage{graphicx}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

\usepackage{color} 

% Use doublespacing - comment out for single spacing
%\usepackage{setspace} 
%\doublespacing


% Text layout
\topmargin 0.0cm
\oddsidemargin 0.5cm
\evensidemargin 0.5cm
\textwidth 16cm 
\textheight 21cm

% Bold the 'Figure #' in the caption and separate it with a period
% Captions will be left justified
\usepackage[labelfont=bf,labelsep=period,justification=raggedright]{caption}

% Use the PLoS provided bibtex style
\bibliographystyle{plos2009}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother


% Leave date blank
\date{}

\pagestyle{myheadings}
%% ** EDIT HERE **


%% ** EDIT HERE **
%% PLEASE INCLUDE ALL MACROS BELOW

%% END MACROS SECTION

\begin{document}

% Title must be 150 characters or less
\begin{flushleft}
{\Large
\textbf{openSNP - Crowdsourcing Genome Wide Association Studies}
}
% Insert Author names, affiliations and corresponding author email.
\\
Bastian Greshake$^{1,\ast}$, 
Philipp Bayer$^{2}$, 
Fabian Zimmer$^{3}$,
Julia Reda$^{4}$
\\
\bf{1} Bastian Greshake Frankfurt am Main, Germany
\\
\bf{2} Philipp Bayer somewhere, Australia
\\
\bf{3} Fabian Zimmer M\"unster, Germany
\\
\bf{4} Julia Reda, Mainz, Germany
\\
$\ast$ E-mail: info@opensnp.org
\end{flushleft}

% Please keep the abstract between 250 and 300 words
\section*{Abstract}

% Please keep the Author Summary between 150 and 200 words
% Use first person. PLoS ONE authors please skip this step. 
% Author Summary not valid for PLoS ONE submissions.   
\section*{Author Summary}

\section*{Introduction}
Genome Wide Association Studies (GWAS) are an easy and cheap way to find Single Nucleotide Polymorphisms (SNPs) which can be interesting because of their medical relevance. SNPs found through GWAS can be used to find candidate genes for a closer inspection or to predict disease risks. Genome Wide Association Studies make use of statistics to compare the alleles of patients to the alleles of healthy controls. By this the method does not allow to find causal differences but mere correlations. The first GWAS was published in 2005 and compared age-related macular degeneration in contrast to a healthy control group (doi:10.1126/science.1109557). Since the beginning the number of participants in those studies is rising and over 1200 GWAS have been performed (doi:10.1186/1471-2350-10-6.) and over 5000 SNPs have been linked to different diseases and traits in those studies %(http://www.genome.gov/page.cfm?pageid=26525384&clearquery=1#result_table).

Since 2006 there are different companies like 23andMe, deCODEme or FamilyTreeDNA on the market which offer Direct-To-Consumer (DTC) genetic testing. Those companies use DNA micro arrays to screen for around 1 million SNPs which are spread over the human genome. In return customers get an analysis of the results, as well as a raw file that includes the SNP-IDs and their respective allele for the customer. In 2011 23andMe alone had over 100.000 customers (http://spittoon.23andme.com/2011/06/15/23andme-2011-state-of-the-database-address/) and the company also recognizes the potential to do GWAS with that amount of data. They provide surveys to their customers that ask about traits and genetic diseases. With the consent of the customer those data will then be used for association studies. 23andMe published 3(?) articles in 2011 in which they replicate known findings but also find new associations for Parkinson's Disease. By activating their customer base they achieved to have over 30.000 individuals enrolled in those association studies.  

Although companies like 23andMe are willing to contribute to science it is not easy for individual scientists to get hold of the data. This arises mainly due to privacy concerns of the customers. Nevertheless there are individual customers who are willingly sharing their data. Most do though by uploading it to their personal website or to software repositories like GitHub. While this is makes it possible to use the data, it requires a lot of work to keep track of all new genotyping data that is available to the public. While projects like the SNPedia try to keep track of all the files, this still does not allow to perform GWAS, as the phenotypic information is not attached to the genetic information. Projects that attach the phenotype to the genetic information, like the Personal Genome Project, still don't allow for an easy re-use of the data.  

A possible solution to this can be a community-driven platform that aggregates genetical and phenotypical information of people who are willing to share their data with the general public and have given their informed consent. In our study we investigated if there would be interest in such a crowd sourcing platform, how many people would be willing to share their genetic and phenotypic information with the public and built such a platform. 

% Results and Discussion can be combined.
\section*{Results}

\subsection*{Survey on Sharing Genetic Information}
229 people, 180 with a self-reported chromosomal sex of XY, 56 with a self-reported chromosomal sex of XX, participated in the survey. The mean age of the participants is 33 (SD = 11,29) and over 81.7 \% reported their ethnicity as caucasian. 39.7 \% of the participants are already customer of at least one DTC genetic testing company and further 30.1 \% of them plan to become one in the future. 29.7 \% don't plan to become a DTC customer. There is no significant difference in the usage of DTC companies between chromosomal sexes (Somers-d). 

67.7 \% of all participants would share their data with their DTC-company without any constraints, 25.8 \% would do so, if the company does not share the data with third parties. 6.6 \% of the participants would not share their data. There is no significant difference between sharing-habits between both chromosomal sexes (Somers-d). Those who are a customer of a DTC company or are planing to become one in the future are more likely to share their results, compared to those who don't plan to get themselves genotyped (Somers-d). 

There are significant differences between those people who are already genotyped and those who don't plan to get genotyped: (All those numbers are tukey-hsd test). The first group is more likely to agree to share their information because they want to help scientists (mean difference = 0.465, SE = 0.128, p = 0.001), because they think of personal benefits (mean difference = 0.448, SE = 0.183, p = 0.04) and because they are curious (mean difference = 1.159, SE = 0.193, p < 0.001). 

On the other hand those hand those people who are not planning to get genotyped are more likely to not share their data, because they agree to fear discrimination (mean difference = 1.060, SE = 0.195, p < 0.001), because they agree that they feel it is a breach of their privacy (mean difference = 0.821, SE = 0.225, p = 0.001), because agree that they fear negative consequences for their family (mean difference = 0.733, SE = 0.21, p = 0.002) or because they agree that they fear personalized advertising (mean difference = 0.848, SE = 0.208, p < 0.001).

Similarly those people who would share data with their DTC provider are more likely to agree on sharing the data, because they want to help scientists (mean difference = 1.57, SE = 0.199, p < 0.001), because they think of personal benefits (mean difference = 0.951, SE = 0.308, p = 0.006), and because they are curious (mean difference = 1.99, SE = 0.321, p < 0.001). 

Those participants who are not planning to get genotyped are more likely to agree to not share their data, because they fear discrimination (mean difference = 1.52, SE = 0.322, p < 0.001), because they feel it is a breach of their privacy (mean difference = 1.871, SE = 0.324, p < 0.001), because they fear consequences for their family (mean difference = 1.146, SE = 0.32, p = 0.001) and because they fear personalized advertising (mean difference =  1.112, SE = 0.357, p = 0.006). 
\subsection*{Subsection 2}

\section*{Discussion}

% You may title this section "Methods" or "Models". 
% "Models" is not a valid title for PLoS ONE authors. However, PLoS ONE
% authors may use "Analysis" 
\section*{Materials and Methods}

% Do NOT remove this, even if you are not including acknowledgments
\section*{Acknowledgments}


%\section*{References}
% The bibtex filename
\bibliography{template}

\section*{Figure Legends}
%\begin{figure}[!ht]
%\begin{center}
%%\includegraphics[width=4in]{figure_name.2.eps}
%\end{center}
%\caption{
%{\bf Bold the first sentence.}  Rest of figure 2  caption.  Caption 
%should be left justified, as specified by the options to the caption 
%package.
%}
%\label{Figure_label}
%\end{figure}


\section*{Tables}
%\begin{table}[!ht]
%\caption{
%\bf{Table title}}
%\begin{tabular}{|c|c|c|}
%table information
%\end{tabular}
%\begin{flushleft}Table caption
%\end{flushleft}
%\label{tab:label}
% \end{table}

\end{document}

